packages/protoplugin-example/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "protoc-gen-twirp-es",
+  "version": "0.1.0",
+  "description": "",
+  "scripts": {
+    "build": "../../node_modules/typescript/bin/tsc --noEmit",
+    "app:buf:generate": "buf generate buf.build/bufbuild/eliza",
+    "app:start": "npm run app:buf:generate && npm run app:serve",
+    "app:serve": "../../node_modules/typescript/bin/tsc --noEmit && npx esbuild src/index.ts --serve=localhost:3000 --servedir=www --outdir=www --bundle --global-name=eliza"
```

Comment:
Yes, adding `generate` and `build` to the Makefile is valuable because `tsc` will catch errors `eslint` does not, and running the plugin will surface runtime errors that could slip through.

Reply:
Sounds good.  Added them to the `make` process.

I added `$(BUILD)/protoplugin` as a prerequisite, but wasn't sure if `protobuf` should also be.  Related to this, I noticed that `$(BUILD)/protoplugin` is not a prereq of the `protoplugin-test` package.  Not sure if it should be?

Reply:
To get a sense of the necessary prerequisites, take a look at packages/protoplugin-example/package.json:

```json
  "dependencies": {
    "@bufbuild/protobuf": "^0.1.1",
    "@bufbuild/protoc-gen-es": "^0.1.1",
    "@bufbuild/protoplugin": "^0.1.1",
    // ...
  }
```

All three dependencies are part of the monorepo, so `npm install` is not sufficient to make them available, we also need to build them. 

Looking at the scripts we run when we build the example, we really use all three of them, so we do need all of them available.

That means `$(BUILD)/protobuf $(BUILD)/protoc-gen-es $(BUILD)/protoplugin` is a proper prerequisite.

I am okay with just using `$(BUILD)/protoc-gen-es` though, because the other two are prerequisites of it.

Reply:
I was weighing this too, and came to the same thought that it's fine to use a single target here.

`packages/protoplugin-example/buf.gen.yaml` should be a prerequisite though.